src/main/java/uk/gov/hmcts/divorce/legaladvisor/event/LegalAdvisorMakeServiceDecision.java

```diff
@@ -163,16 +163,16 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(
                         DISPENSED_WITH_SERVICE_REFUSED_FILE_NAME,
                         DISPENSE_WITH_SERVICE_REFUSED,
                         SERVICE_REFUSAL_TEMPLATE_ID);
+                    endState = caseDataCopy.getApplication().getIssueDate() != null ? AwaitingAos : ServiceAdminRefusal;
                 } else if (DEEMED.equals(serviceApplication.getAlternativeServiceType())) {
                     generateAndSetOrderToDeemedOrDispenseDocument(
                         caseDataCopy,
                         details.getId(),
                         DEEMED_SERVICE_REFUSED_FILE_NAME,
                         DEEMED_SERVICE_REFUSED,
                         SERVICE_REFUSAL_TEMPLATE_ID);
+                    endState = AwaitingAos;
                 }
-
-                endState = AwaitingAos;
             }
```

Comment:
should there still be a default endState set, maybe ServiceAdminRefusal?

Reply:
As mentioned in Slack comment, there are only two possible services for the state this event is available for (Dispensed and Deemed). Third option is Bailiff service which has other events available to progress with it and because of that there will definitely be a state from one of ServiceAdminRefusal, AwaitingAos when service is refused by LA.